test/randomized/test_transpiler_equivalence.py

```diff
@@ -13,18 +13,36 @@
 
 """Randomized tests of transpiler circuit equivalence."""
 
+import os
 from math import pi
 
+from distutils.util import strtobool
 from hypothesis import assume, settings, HealthCheck
 from hypothesis.stateful import multiple, rule, precondition, invariant
 from hypothesis.stateful import Bundle, RuleBasedStateMachine
 
 import hypothesis.strategies as st
 
+from hypothesis import settings
+
```

Comment:
```suggestion
```

(duplicated import)

Reply:
I think there's a couple of other places where lint might be awkward, e.g. I'm not sure it lets you put code inbetween imports.

Reply:
Done in bdff868c3ac5ad0d85be74fe2ee95b54f72a2481.